README.md

```diff
@@ -122,60 +106,10 @@ If you have dataset with over 10 subjects and since each defacing job is indepen
 subject/session in the dataset using the `-n/--n-cpus` option. The following example command will run the pipeline occupying 10 processors at a time.
 
 ```bash
-python src/dsst_defacing_wf.py ${INPUT_DIR} ${OUTPUT_DIR} -n 10
+python src/run.py ${INPUT_DIR} ${OUTPUT_DIR} -n 10
```

Comment:
please make this a 5-line code block where the first line is conda activate. something like:

```suggestion
# activate your conda environment
conda activate

# once your conda environment is active, execute the following
python src/run.py ${INPUT_DIR} ${OUTPUT_DIR} -n 10
```